assets/js/DraggableHeaderCell.js

```diff
@@ -0,0 +1,100 @@
+// Pasted from react-data-grid according to this comment:
```

Comment:
This file should be installed with react-data-grid-addons https://github.com/adazzle/react-data-grid/tree/master/packages/react-data-grid-addons